tests/e2e/tests/test_profile_irsa.py

```diff
@@ -163,7 +163,7 @@ def profile_trust_policy(
                 "Action": "sts:AssumeRoleWithWebIdentity",
                 "Condition": {"StringEquals": {
                     f"{oidc_url}:aud": "sts.amazonaws.com",
-                    f"{oidc_url}:sub": [f"system:serviceaccount:profile-aws-iam:default-editor"]
+                    f"{oidc_url}:sub": [f"system:serviceaccount:profile-aws-iam:default"]
```

Comment:
from the instructions on website: https://awslabs.github.io/kubeflow-manifests/docs/component-guides/profiles/ only `default-editor` service account will be annotated. can we try one of the options below so that the `default-editor` sa can be used by katib as well? We need to document the same

looks like katib allows [customizing service account used by experiment](https://github.com/kubeflow/katib/pull/1092). Also documented here: https://www.kubeflow.org/docs/components/katib/katib-config/#suggestion-settings.

another option is to specify serviceAccountName in the trial spec sample [here](https://github.com/kubeflow/katib/blob/7be1f0ad5730ab6216570d2179f5b23f0b39d26d/examples/v1beta1/argo/argo-workflow.yaml#L46). I dont know if all trial types allow this field

Reply:
Incorporated this into my latest changes below